lib/rules/common/tap-synonyms.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+// @see https://github.com/tapjs/node-tap/blob/f323cdca28cc0a286e29585d8081e04b08704143/lib/synonyms.js
+// A list of all the synonyms of assert methods.
+// In addition to these, multi-word camelCase are also synonymized to
+// all lowercase and snake_case
+
+function multiword(obj) {
+  return Object.keys(obj).reduce((s, i) => {
+    s[i] = [multiword_(i)]
+      .concat(obj[i].map(multiword_))
+      .reduce((s, i) => {
+        s.push.apply(s, i)
```

Comment:
you don't need the apply you can just `s.push(s, i)`

Reply:
ack - also I didn't make this clear, but this is just copied from tap

Reply:
Unless i is an array? then it would need to be `apply`

Reply:
indeed, it's an array